src/cascadia/CascadiaPackage/NOTICE.html

```diff
@@ -0,0 +1,16 @@
+<html>
+  <head><title>Placeholder 3rd-party Notices</title></head>
+  <body>
+    <h1>Windows Terminal (Dev)</h1>
+    <p>
+      This is a development build of Windows Terminal. The third-party notices
+      for this project can be found on
+      <a href="https://github.com/microsoft/terminal/blob/master/NOTICE.md">the
+      project's GitHub page</a>.
+    </p>
+    <p>
+      During a branded release build, this file is replaced with the content of
+      <tt>NOTICES.md</tt> from the root of the repository.

```

Comment:
woah I've never seen `<tt>` before - shouldn't it be `<code>` though? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tt

Reply:
I have no idea which is more correct to use. :)

Reply:
```suggestion
      <code>NOTICES.md</code> from the root of the repository.
```